transverb/transverbformalities.cpp

```diff
@@ -70,6 +71,7 @@ Transverb::Transverb(TARGET_API_BASE_INSTANCE_TYPE inInstance)
   addparameterattributes(kDist1, DfxParam::kAttribute_OmitFromRandomizeAll);
   addparameterattributes(kDist2, DfxParam::kAttribute_OmitFromRandomizeAll);
   addparameterattributes(kFreeze, DfxParam::kAttribute_OmitFromRandomizeAll);
+  addparameterattributes(kAttenuateFeedbackByMixLevel, DfxParam::kAttribute_OmitFromRandomizeAll);
 
   std::vector<long> mixparameters(1, kDrymix);
   for (size_t head = 0; head < kNumDelays; head++) {
```

Comment:
I dunno why I can't comment on it, but line 200 doesn't compile for me:

```
heads[h].read = fmod_bipolar(static_cast<double>(writer) - (*dist * bsize_f), bsize_f);
```

it should be heads[head] I think? Bad merge or something?

Reply:
IIRC there was also a very old feature of C++ where the scope of a variable like h in `for (int h; ...)` would extend past the body of the `for` loop, so it's possible that if this is mysteriously compiling for you, for some reason your setup is still allowing that (not recommended for obvious reasons :))

Reply:
Oops yes I got some commit mixup happening in my attempts to have multiple changes going on at the same time and doing messy plucking of bits. Should be fixed now!
I think that for loop declaration extension issue was a C89 thing?

Reply:
oops actually I had busted that on main, so I fixed it there too and then rebased this branch on main.

Reply:
Yeah, it might have even predated C++. I remember MSVC supporting it for a long time after it was standardized away. But of course moot if this was just a bad merge.

This change looks good to me now! Everything still works intuitively and the presets sound like I remember 'em, so I think this was just a bug; thanks for finding and fixing.